build-support/docker/remote_execution/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+FROM gcr.io/cloud-marketplace/google/rbe-ubuntu16-04@sha256:da0f21c71abce3bbb92c3a0c44c3737f007a82b60f8bd2930abc55fe64fc2729
```

Comment:
Do we want to base on this, or on some other image that we are more familiar with? Is there value in figuring out what the base for travis's image is, and starting there instead?

Reply:
I think that it makes sense to use this as the basis. From what I understand, Google intentionally gave it a limited set of powers with the intention of keeping things as hermetic and simple as possible. In contrast, the Travis images seem to have things irrelevant to us, like `STOPSIGNAL SIGINT`: https://github.com/travis-ci/worker/blob/master/Dockerfile.